smartsim/_core/config/config.py

```diff
@@ -236,6 +236,12 @@ def test_account(self) -> t.Optional[str]:  # pragma: no cover
         # no account by default
         return os.environ.get("SMARTSIM_TEST_ACCOUNT", None)
 
+    @property
+    def test_no_mpi(self) -> bool:  # pragma: no cover
+        # None means that an MPI app will be tested if a small
+        # test program compiles: anything else will disable it
+        return "SMARTSIM_TEST_NO_MPI" in os.environ
```

Comment:
Small suggestion to implimentation just to stay consistent with `SMARTSIM_FLAG_TELEMETRY`
```suggestion
        return int(os.environ.get("SMARTSIM_TEST_NO_MPI", "0")) > 0
```

Reply:
OK, did the same, but set default to `1` - implying we _try_ to run it.

Reply:
Ahh, I just saw that you fixed this before I even posted the review! LGTM!!